pkg/datarepair/checker/checker_test.go

```diff
@@ -35,6 +37,11 @@ var ctx = context.Background()
 func TestIdentifyInjuredSegments(t *testing.T) {
 	logger := zap.NewNop()
 	pointerdb := pointerdb.NewServer(teststore.New(), &overlay.Cache{}, logger, pointerdb.Config{}, nil)
+	assert.NotNil(t, pointerdb)
+
+	sdb, err := statdb.NewStatDB("sqlite3", fmt.Sprintf("file:memdb%d?mode=memory&cache=shared", rand.Int63()), logger)
```

Comment:
you can also get statdb pretty easily from a testplanet setup. Example here if you're interested in that: https://github.com/storj/storj/blob/master/pkg/overlay/cache_test.go#L94-L100